pyproject.toml

```diff
@@ -1,20 +1,6 @@
-[tool.poetry]
```

Comment:
Will be still be able to create SHA pinned requirements.txt without these poetry bits?